gammapy/utils/time.py

```diff
@@ -0,0 +1,30 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Time related utility functions."""
+from astropy.time import Time
+
+__all__ = ['time_ref_from_dict',
+           ]
+
+
+def time_ref_from_dict(meta):
+    """ Calculte the time reference from metadata.
+    The time reference is built as MJDREFI + MJDREFF in units of MJD.
+    All other times should be interpreted as seconds after the reference.
+
+    Parameters
+    ----------
+    meta : dict
+    	dictionnary with the keywords 'MJDREFI' and 'MJDREFF'
+
+    Returns
+    -------
+    `~astropy.Time` object with the reference time in units of MJD.
```

Comment:
Does the formatting look OK here in the html docs output?

The Astropy time lingo is "format" for MJD. Please use that here instead of "in units of MJD".